### PR TITLE
Add CD Baby

### PR DESCRIPTION
**Pull request by adeishs**

### CODE REVIEW

File: _data/entertainment.yml [hunk 1]
```diff
@@ -1,4 +1,10 @@
 websites:
+    - name: CD Baby
+      url: http://www.cdbaby.com/
```
Comment (Carlgo11) [inline _data/entertainment.yml]: Please change the url to `https://www.cdbaby.com/`

Comment (adeishs) [inline _data/entertainment.yml]: OK. Done.

**Comment by mxxcon** [comment]
@adeishs please rebase your PR with the latest version so that it can be cleanly merged.

**Comment by jamcat22** [comment]
Thanks for the PR @adeishs! Even though your PR has been merged, it might take some time before it shows up on the website. For more information, check out our [wiki](https/::github.com/2factorauth/twofactorauth/wiki/FAQ_MERGE).

**Comment by adeishs** [comment]
No worries, @jamcat22! Thanks.

**Pull request merged**